packages/plugin-ext/src/typings/monaco/index.d.ts

```diff
@@ -14,7 +14,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-/// <reference types='monaco-editor-core/monaco'/>
+/// <reference types='@typefox/monaco-editor-core/monaco'/>
```

Comment:
maybe it's out of scope here, but wouldn't it make sense to reference `@theia/monaco` here?

Reply:
it is out of scope, please file an issue